src/averageAges.js

```diff
@@ -1,3 +1,4 @@
+
 'use strict';
 
 /**
```

Comment:
Avoid Repetition: Created a helper function filterPeople to avoid redundancy in the filtering logic.

`const filterPeople = (people, filterFn) => people.filter(filterFn);`

And use it in the functions:
- calculateMenAverageAge
- calculateWomenAverageAge
- calculateAverageAgeDiff